### PR TITLE
adding quarkus for bootstrap

### PR DESCRIPTION
**Pull request by dandreadis**

### CODE REVIEW

**Comment by dandreadis** [comment]
With help from @ebullient & @maxandersen.

**Comment by kenfinnigan** [comment]
Hi @dandreadis, there are some required changes missing from the PR. Could you also update:

- CONTACTS.yaml
- PROJECTS.yaml
- TRADEMARKS.md

Thanks

**Comment by maxandersen** [comment]
@kenfinnigan my bad - missed those in my review and @dandreadis is travelling atm. I've pushed the files so you can verifyu/validate and then @dandreadis can squash and sign so he confirms it as Quarkus EGC rep.

**Review by ebullient — APPROVED** [review]

File: TRADEMARKS.md [hunk 1]
```diff
@@ -30,6 +30,9 @@ The Commonhaus Foundation uses the following unregistered trademarks:
 - OpenRewrite &trade; [^1]
 - SDKMAN! &trade; [^1]
 - Feign &trade; [^1]
+- Quarkus &trade; [^1]
+
+[^1]: Trademark is used by permission subject to the foundation's [bootstrapping agreement](
```
Comment (kenfinnigan) [inline TRADEMARKS.md:35]: I think this footnote line was added by mistake, could you remove it? Thanks

Comment (ebullient) [reply TRADEMARKS.md:35]: oh! right. This doesn't need to be here (as it is at the bottom).
```suggestion
```

**Review by ebullient — CHANGES_REQUESTED** [review]

File: TRADEMARKS.md (hunk 1 above)
Comment (maxandersen) [inline TRADEMARKS.md:35]:
```suggestion
```

**Review by ebullient — APPROVED** [review]

**Comment by dandreadis** [comment]
Are all reviewed changes in now? Thanks folks!

**Comment by ebullient** [comment]
> Are all reviewed changes in now? Thanks folks!

Yes. Internal discussions now (project review and agreement).

**Review by cealsair — APPROVED** [review]

**Pull request merged**